source/tutorials/learning-paths/pipelines/_intro-pl.rst

```diff
@@ -0,0 +1,10 @@
+.. admonition:: Note
+    :class: seealso
+
+    This tutorial is a part of our :ref:`learning-paths-pipelines` series. Make sure you have completed the :ref:`learning-paths-fundamentals` learning path first. 
+
+.. admonition:: Prerequirements
+    :class: attention
+
+    * :ref:`repository` before using pipelines.
```

Comment:
Is a Git repository really a requirement, or just strongly recommended? It used to be a must but we might want to still do this learning path with `--adhoc`

Reply:
I kind of had the impression that we would like to push users to connecting to Git repos when using pipelines. There is a section under running pipelines on `--adhoc` pipelines but I didn't want to emphasize that too much. 

I agree with your later comment that we seem to have trials where users are not familiar with Git so it does also make sense to have the tutorial without needing to connect to a repo. In the end, I don't have any strong opinions so I can change this to focus on `--adhoc` first. :)